Support documents

gcaba3 opened the pull request.

gcaba3 committed:
Task Assignment  Sheet

See which file is assigned to who

gcaba3 committed:
Updated With sprint two plans

gcaba3 committed:
Revised Sprint plan and user stories

I changed most accpetance criteria, to make them easier to test. Also
split some big tasks into two smaller tasks. While adding some new ones
in, like a way to register a non-members so they can attend events.

gcaba3 committed:
Mainly just adding burndown chart

Added current sprint and total tasks, to help with the burndown chart

gcaba3 committed:
Mostly finished profile management

Cant get reentering the password to work

gcaba3 committed:
Updated for todays work

Miscalculated how much tasks we really had in the beginning but, the
graph still shows the same general pattern

gcaba3 committed:
Basic Acceptance tests

gcaba3 committed:
Most accetance tests done, except one

Still no automation testing used.

gcaba3 committed:
General update

gcaba3 committed:
Updated for todays work

gcaba3 committed:
Finalised with release one work

benwatton92 commented:
[Automated testing PHP.docx](https://github.com/gcaba3/IFB299-Group23/files/479331/Automated.testing.PHP.docx)

gcaba3 committed:
The support docs for sprint 3

Dont get intimidated by the large amount of tasks. Most of it comes
styling.

Essentially what i was thinking was making the already existing website
as a homepage for none members. Then as registered users (member,
planner, volunteer) uses these webpages after the login.

This time around were gonna build each page one by one. I know it might
be slow, but its easier to divide the work up this way. Most of the
tasks and user stories should be modulor, as in you can build the page
for a tab, then over time just add the links to allow it to connect to
other pages. Also you have to apply styling which havent done, so for
now just focus on creating functionality, displaying the required info,
forms with textboxes, sql queries are working etc.

If you want a storyboardish/overview dataflow type diagram go to my
personal portfolio or google drive under release 2, to find the designs.
Thats where i based these tasks on, most user stories are one tab.

gcaba3 committed:
Just noticed there wasnt a table for some tasks

gcaba3 committed:
FInished the login feature, still gotta do account page

Also theres a template there. You can just copy that and edit for
whatever tab your working on. It should carry the CSS stylings

gcaba3 committed:
Finished: Edit login details and account details

Signed on to do register new member

gcaba3 committed:
Just added dates

gcaba3 committed:
FInished: register new member

Signed on to do event viewing event.

gcaba3 committed:
General Update

added lances name on the work he assigned himself.
Also moved events attending task to this sprint, as it is needed to view
the event themselves.

gcaba3 committed:
Moved events attending page

gcaba3 committed:
Revert "Moved events attending page"

This reverts commit bb55ff27aa3785a8a574227069e00a954d148f4c.

gcaba3 committed:
Revert "Revert "Moved events attending page""

This reverts commit 1b666387916a6799e8bde6542f4ac59905ffb53f.

gcaba3 committed:
Forgot to delete the taken time

gcaba3 committed:
Forgot to add date completed
